packages/shadydom/src/patch-events.js

```diff
@@ -271,6 +272,31 @@ let EventPatches = {
   },
 };
 
+// Some browsers have missing or broken descriptors. If this is the case
+// eventPhase can't be polyfilled. For example, Chrome 41 does not
+// have prototypical descriptors on DOM objects and Safari 9 has broken ones
+// that cannot be used.
```

Comment:
Don't think it's worth it since this affects only these really old browsers and `eventPhase` generally isn't critically important. We'd likely need to do the manual `dispatchEvent` always on these browsers to fix it, and that doesn't seem worth it.